plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -156,6 +159,7 @@ const toClientSafePodMetrics = (
 type responseWithMetrics = [FetchResponseWrapper, PodStatus[][]];
 
 export class KubernetesFanOutHandler {
+  private readonly EMPTY_SET = new Set<ObjectToFetch>();
```

Comment:
Man, this makes me nervous :P if some consumer (like the fetcher) starts modifying it assuming that that's OK, affecting all subsequent callers... Probably not an optimization that's worthwhile